flutter-winit/src/window.rs

```diff
@@ -184,6 +186,7 @@ impl FlutterWindow {
             localization.send_locale(locale_config::Locale::current());
         });
 
+        let mut active_devices: HashMap<i32, DeviceId> = HashMap::new();
```

Comment:
Can we move `active_devices` inside `FlutterEngine`? As we know its will always cause a crash, we may as well give everyone the sanity check

Reply:
We can also switch to a `hashset` when we move, as we only care about the index, not the actual device id.

Reply:
I'll see if I can switch it over to FlutterEngine. I'll switch over to hashset.

Reply:
Actually the crash happens on interaction with the window. We could have each window manage its own devices.

Reply:
My thoughts are more of, no matter how you're using `FlutterEngine`, sending `send_pointer_event` for a non existent device, or creating two devices will lead to a crash. Why make each window provider duplicate that logic? We can also expose a `is_device_active(i32)` to allow windows to check if a device is active, if needed